chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
+	this.populateLocaleList = function(menulist, prefLocale) {
+		if(!_initialized) this.init();
+		
+		var doc = menulist.ownerDocument;
+		var popup = doc.createElement('menupopup');
+		menulist.appendChild(popup);
```

Comment:
This would allow calling `populateLocaleList` again if, say, for whatever reason, we want to change the `prefLocale` after the window is already opened.